meshnets/utils/data_visualization.py

```diff
@@ -24,6 +23,19 @@ def plot_mesh(mesh_path: str,
               screenshot=screenshot)
 
 
+def _validate_scalars(scalars: np.array) -> np.array:
+
+    if scalars is None:
+        raise TypeError("Argument 'scalars' is None.")
+
+    if scalars.ndim == 1:
+        return scalars
+    elif scalars.ndim == 2 and scalars.shape[1] == 1:
+        return scalars.flatten()
+    elif scalars.ndim > 1:
+        raise ValueError("Argument 'scalars' must have a single dimension.")
```

Comment:
I think this should be an `else` clause (or even better, the clause should be omitted). The message itself is also not accurate, since by line 33 `scalars.ndim` can be 2. Rephrase the error message.

Reply:
You're right, I changed it to an else clause and changed the error message. Why do you think the clause should be omitted? I wrote this method because pyvista allows faulty plots when the provided array does not conform to their required dimensions. This leads to plots that looks plausible but do not represent the correct data.